_php/_objects/class_dropdown_do.php

```diff
@@ -0,0 +1,24 @@
+<?php
+function semSelect{
+$sql = "SELECT SemesterID, SemesterName, Year From semester";
+$semSelect = mysqli_query($con, $sql);
+$all_rows = array();
+while($row=mysql_fetch_array($semSelect){$all_rows[]=$row;}
+return $all_rows;
+}
+
+?>
+
+<?php
+$rows=$dropdo->semSelect();
+foreach ($rows as $ddo) {
+  echo ($ddo == $SemesterID) ? "<option selected=\"selected\" value=\"$ddo\">$ddo</option>" : "<option value=\"$ddo\">$ddo</option>";
+}  
+ ?> 
+  
+<select name="Semester ID">
```

Comment:
Can we safely remove this?

Reply:
I will check on this.

Reply:
Yes, This can be removed. I renamed the file to drop_do.php to include all of them. This one was uploaded by mistake. echo '<sorry>';